src/pages/open-source-for-non-developers.md

```diff
@@ -0,0 +1,31 @@
+# Open Source for Non-Developers
+
+## What is Open Source?
+
+Open source is software whose source code can be freely viewed and redistributed.
+
+Unlike products that contain proprietary code, open source code can be viewed by developers worldwide.
+
+It also allows people to propose changes, which fosters a collaborative working environment.
+
+## Why do Products Become Open Source?
+
+1. To interact with consumers: By making the designs and code open source, consumers have a more direct way to improve the products they enjoy,
+   which in turn can save your company time and effort.
+2. To reach more demographics: Open source products are often freely viewed by many consumers. This can help increase brand awareness.
+3. To build loyalty and trust: Since open source products allow people to see the source code. This creates transparency, which helps build trust between consumers and the product’s creators.
+
+## Why do companies often choose to consume open source?
+
+1. Innovation: Since open source software allows people to propose changes, it allows participation from a wide variety of contributions,
+   which means more contributors can propose ideas and improve the project.
+1. Cost-effective: Open source software is often free, which in turn helps companies save money. This is because the software is developed and maintained by a community of developers, rather than by a single company. This means that there is no need to pay for software licenses, which can be a significant expense for businesses.
+1. Time-saving: Open source software can also help businesses to save time and resources.
+   This is because the software is already developed and tested, so businesses do not need to spend time and money developing their own software.
+   Additionally, open source software can give businesses more time to focus on their core business activities.
+
+## Why do companies often choose to publish open source?
+
+1. To build a community: The transparency of open source products allows companies various ways to promote their products ranging from blog posts about the products to social media posts that spotlight members for their contributions to the product.
+2. Increased security: Since open source communities tend to be large, bugs and tech issues can be fixed in a quick timeframe. According to a [2021 survey done by Red Hat](https://www.redhat.com/en/resources/state-of-enterprise-open-source-report-2022), 89% of IT leaders at companies worldwide believed open source software was just as secure or more secure than proprietary software.
```

Comment:
```suggestion
2. Increased security: Since open source communities tend to be large, bugs and tech issues can be fixed in a quick timeframe. 
According to a [2021 survey done by Red Hat](https://www.redhat.com/en/resources/state-of-enterprise-open-source-report-2022), 89% of IT leaders at companies worldwide believed open source software was just as secure or more secure than proprietary software.
```